jetson/percep/pcl.cpp

```diff
@@ -405,19 +309,23 @@ bool PCL::CheckPath(std::vector<std::vector<int>> interest_points,
     #if PERCEPTION_DEBUG
     //Project path in viewer
     pcl::PointXYZRGB pt1;
-    pt1.x = leftLine.b;
+    pt1.x = leftLine.xIntercept;
     pt1.y = 0;
     pt1.z = 0;
     pcl::PointXYZRGB pt2;
-    pt2.x = rightLine.b;
+    pt2.x = rightLine.xIntercept;
     pt2.y = 0;
     pt2.z = 0;
     pcl::PointXYZRGB pt3(pt1);
     pt3.z=7000;
-    pt3.x=leftLine.m*pt3.z+leftLine.b;
+    pt3.x = leftLine.xIntercept;
+    if(leftLine.m != 0)
```

Comment:
Curly braces are missing (happens a couple other times as well).

Reply:
What part needs curly braces?

Reply:
Around the `if` body. I know you don't _need_ them but you should have them.

Reply:
I thought that was personal preference. Is it generally more professional to include them? I think it's clearer without the curly braces personally

Reply:
Check out [this post](https://stackoverflow.com/questions/2125066/is-it-a-bad-practice-to-use-an-if-statement-without-curly-braces). In particular, check out the first answer and look at the article in links to regarding an infamous apple bug that could have been avoided by always using curly braces.